xprocspec-runner/pom.xml

```diff
@@ -24,7 +24,7 @@
     <dependency>
       <groupId>org.daisy.xprocspec</groupId>
       <artifactId>xprocspec</artifactId>
-      <version>[1.0.0,2.0.0)</version>
+      <version>1.1.1-SNAPSHOT</version>
```

Comment:
It includes your log branch, yes. I'll try to get something released this week. Will work on it now.